src/sortStudents.ts

```diff
@@ -1,16 +1,64 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averageGrade',
```

Comment:
Do you have such key in your student object??
```suggestion
  AverageGrade = 'grades',
```